buddychatcli/Person.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+
+using CommandLine;
+
+namespace BuddyChatCLI
+{
+
+    public class Person
+    {
+        
+        public string username { get; set; }
+
+        public string name { get; set; }
+
+        public string pronoun { get; set; }
+
+        public bool audioChatOnly { get; set; }
+
+        public string intro { get; set; }
+
+        public Dictionary<string, string> questionAndAnswers { get; set;}

```

Comment:
I was thinking this would be more generic like - kvpData. And you would have: 
question 1 = <question 1 text>
answer 1 = <answer 1 text>
...

I think this does mean that we need a config file that converts from the column titles to property names. 
I.e. pairs of:
column="Give a short introduction of yourself to your partner."
shortname="Introduction"

column = "Copy and paste in the first question you picked"
shortname="question1"